src/SelfServe/Documentation/README.md

```diff
@@ -0,0 +1,296 @@
+# Self Serve Model
+
+The Self Serve Model allows you to write classes that auto generate UI components for your feature. The idea is to allow developers from other feature teams, who may not be familiar with writing UI, to develop and own UX components. This is accomplished by just writing simpler TypeScript classes for their features. 
+
+What this means for the feature team 
+- Can concentrate just on the logic behind showing, hiding and disabling UI components 
+- Need not worry about specifics of the UI language or UX requirements (Accessibility, Localization, Themes, etc.)
+- Can own the REST API calls made as part of the feature, which can change in the future
+- Quicker turn around time for development and bug fixes since they have deeper knowledge of the feature
+
+What this means for the UI team
+- No need to ramp up on the intricacies of every feature which requires UI changes
+- Own only the framework and not every feature, giving more bandwidth to prioritize inhouse features as well
+
+## Getting Started
+
+Clone the cosmos-explorer repo and run
+
+- `npm install`
+- `npm run build`
```

Comment:
for more info on set up, please check here... and give link to DE's getting started doc.